From 89515183b81df7d1b6e4551fc097a548fbc2cbe0 Mon Sep 17 00:00:00 2001
From: Anish Trivedi <anish@freescale.com>
Date: Thu, 31 Mar 2011 15:10:18 -0500
Subject: [PATCH 215/449] ENGR00141476 [imx-test] SCC2 remove return statement
 from partition_tests.sh

Remove return statement at the end of the script to avoid
error message when the script is not sourced directly. It is
called from autorun-scc2.sh script.

Signed-off-by: Anish Trivedi <anish@freescale.com>
---
 test/mxc_scc2/partition_tests.sh | 1 -
 1 file changed, 1 deletion(-)

diff --git a/test/mxc_scc2/partition_tests.sh b/test/mxc_scc2/partition_tests.sh
index 3c8fb40..8c83060 100644
--- a/test/mxc_scc2/partition_tests.sh
+++ b/test/mxc_scc2/partition_tests.sh
@@ -28,4 +28,3 @@ echo
 
 print_test_results
 
-return
-- 
1.8.0

